R/teal.data.R

```diff
@@ -11,8 +11,9 @@
 
 # Fix R CMD check notes
 #' @import shiny
+#' @importFrom rlang %||%
 #' @importFrom digest digest
 #' @importFrom stats setNames
-#' @importFrom shinyjs show
+#' @importFrom shinyjs useShinyjs
```

Comment:
All declared imports should be used. `useShinyjs` is used inside of R6 so compiler doesn't recognise this. BTW I've made this change only because `shinyjs show` override `utils` generic for `show` method

Reply:
This is why we have `.rectify_dependencies_check` in `teal.slice` [here](https://github.com/insightsengineering/teal.slice/blob/686f1f76ac6130325df9314fea419800c139f35a/R/zzz.R#L37).

Overrides? Not masks? Prefixing doesn't work? :exploding_head:

Reply:
Can't wait :laughing:

Reply:
> Who cares? In two weeks it won't be discussed anymore ;]

Best argument ever : P